### PR TITLE
[#125881031]list_briefs view: add "meta"->"total" attribute to response for returning total result count

### PR DESCRIPTION
**Pull request by risicle**
update tests accordingly

Forgot I would need this for story https://www.pivotaltracker.com/story/show/125881031

### CODE REVIEW

File: app/main/views/briefs.py [hunk 1]
```diff
@@ -136,23 +136,31 @@ def list_briefs():
...
+
     if user_id:
         return jsonify(
             briefs=[brief.serialize() for brief in briefs.all()],
-            links=dict()
+            meta=meta,
+            links={},
         )
     else:
+        total_count = briefs.count()
```
Comment (allait) [inline app/main/views/briefs.py]: I don't think this is used anywhere

File: app/main/views/briefs.py [hunk 2]
```diff
@@ -136,23 +136,31 @@ def list_briefs():
             status.strip() for status in request.args['status'].split(',')
             ))
 
+    meta = {
+        "total": briefs.count(),
```
Comment (allait) [inline app/main/views/briefs.py]: Pagination objects have a [`.total` attribute](http://flask-sqlalchemy.pocoo.org/2.1/api/#flask.ext.sqlalchemy.Pagination.total), so I think we can avoid doing a separate count.

File: app/main/views/briefs.py (hunk 1 above)
Comment (risicle) [inline app/main/views/briefs.py]: Yup this should have got scrubbed

**Comment by TheDoubleK** [comment]
Looks good after a rebase :+1:

**Comment by pcraig3** [comment]
Should we do this for services too?  
@TheDoubleK, @andrewchong and I were counting a lot of services today by figuring out the number of pages and multiplying them by 100.

**Comment by risicle** [comment]
@pcraig3 perhaps, but the services are fetched through the search api, so separate job. Actially I was talking to you about pagination on the search api the other day, wasn't I?

**Pull request merged**